src/stan/services/pathfinder/multi.hpp

```diff
@@ -117,28 +117,33 @@ inline int pathfinder_lbfgs_multi(
       individual_samples;
   individual_samples.resize(num_paths);
   std::atomic<size_t> lp_calls{0};
-  tbb::parallel_for(
-      tbb::blocked_range<int>(0, num_paths), [&](tbb::blocked_range<int> r) {
-        for (int iter = r.begin(); iter < r.end(); ++iter) {
-          auto pathfinder_ret
-              = stan::services::pathfinder::pathfinder_lbfgs_single<true>(
-                  model, *(init[iter]), random_seed, stride_id + iter,
-                  init_radius, history_size, init_alpha, tol_obj, tol_rel_obj,
-                  tol_grad, tol_rel_grad, tol_param, num_iterations,
-                  num_elbo_draws, num_draws, save_iterations, refresh,
-                  interrupt, logger, init_writers[iter],
-                  single_path_parameter_writer[iter],
-                  single_path_diagnostic_writer[iter], calculate_lp);
-          if (unlikely(std::get<0>(pathfinder_ret) != error_codes::OK)) {
-            logger.error(std::string("Pathfinder iteration: ")
-                         + std::to_string(iter) + " failed.");
-            return;
+  try {
+    tbb::parallel_for(
+        tbb::blocked_range<int>(0, num_paths), [&](tbb::blocked_range<int> r) {
+          for (int iter = r.begin(); iter < r.end(); ++iter) {
+            auto pathfinder_ret
+                = stan::services::pathfinder::pathfinder_lbfgs_single<true>(
+                    model, *(init[iter]), random_seed, stride_id + iter,
+                    init_radius, history_size, init_alpha, tol_obj, tol_rel_obj,
+                    tol_grad, tol_rel_grad, tol_param, num_iterations,
+                    num_elbo_draws, num_draws, save_iterations, refresh,
+                    interrupt, logger, init_writers[iter],
+                    single_path_parameter_writer[iter],
+                    single_path_diagnostic_writer[iter], calculate_lp);
+            if (unlikely(std::get<0>(pathfinder_ret) != error_codes::OK)) {
+              logger.error(std::string("Pathfinder iteration: ")
+                           + std::to_string(iter) + " failed.");
```

Comment:
If we are already throwing in the individual pathfinder do we need this message?

Reply:
This can still happen if e.g. a path fails to initialize. That isn't fatal to the entire algorithm, so it will trigger this message, which I think is still valuable.